controllers/storagecluster/cephcluster_test.go

```diff
@@ -466,10 +466,10 @@ func createDummyKMSConfigMap(kmsProvider, kmsAddr string, kmsAuthMethod string)
 		cm.Data["VAULT_BACKEND_PATH"] = "ocs"
 		cm.Data["VAULT_NAMESPACE"] = "my-ocs-namespace"
 	case IbmKeyProtectKMSProvider:
-		cm.Data["IBM_SERVICE_INSTANCE_ID"] = "my-instance-id"
-		cm.Data["IBM_KMS_KEY"] = "my-kms-key"
-		cm.Data["IBM_BASE_URL"] = "my-base-url"
-		cm.Data["IBM_TOKEN_URL"] = "my-token-url"
+		cm.Data["IBM_KP_SERVICE_INSTANCE_ID"] = "my-instance-id"
+		cm.Data["IBM_KP_SECRET_NAME"] = "my-kms-key"
+		cm.Data["IBM_KP_BASE_URL"] = "my-base-url"
+		cm.Data["IBM_KP_TOKEN_URL"] = "my-token-url"
```

Comment:
Add secret name test on:
https://github.com/red-hat-storage/ocs-operator/blob/6835df80bce5186b4d89129c50000e3f303b5204/controllers/storagecluster/cephcluster_test.go#L572
https://github.com/red-hat-storage/ocs-operator/blob/6835df80bce5186b4d89129c50000e3f303b5204/controllers/storagecluster/noobaa_system_reconciler_test.go#L520

Reply:
for Vault we had constant Secret name `ocs-kms-secret` that is what we are asserting in the tests...
but for HPCS this name is not constant, it is randomly generated name... so I think we can skip asserting the Secret name...